Resources/Private/Scss/App.scss

```diff
@@ -29,6 +30,7 @@
 @import 'ezbrequest';
 @import 'find';
 @import 'solr';
+@import 'newAquisitions';
```

Comment:
We really should agree on one scheme… camelCase or hyphen?

Reply:
We should consolidate that in another issue